src/patternfly/components/Tabs/examples/Tabs.md

```diff
@@ -231,6 +231,14 @@ import './Tabs.css'
 {{/tabs}}
 ```
 
+### Vertical, box, expanded
```

Comment:
total nit - We have "Vertical", "Box vertical", and "Vertical, box, expanded"

I wonder if that should be
* "Vertical", "Box vertical", and "Box vertical expanded"
* "Vertical", "Vertical box", and "Vertical box expanded"

Just looking at the rest of the examples, I would assume the latter. We could probably even take "horizontal" off of the example names that have them as that's the default, and prefix all of the vertical examples with "Vertical"?

The exception would be "horizontal overflow", but I dunno if that's the best description for that anyways. The default overflow is horizontal overflow, too, since we don't have vertical overflow. "horizontal overflow" has the overflow tab. Maybe that should be called "Overflow tab"? Just some thoughts, not to hold up or impact this PR.